From: Andrew Morton <akpm@linux-foundation.org>
Subject: writeback-support-retrieving-per-group-debug-writeback-stats-of-bdi-fix
Date: Mon Apr 22 03:45:50 PM PDT 2024

fix build

mm/backing-dev.c: In function 'cgwb_debug_stats_show':
mm/backing-dev.c:195:25: error: unused variable 'stats' [-Werror=unused-variable]
  195 |         struct wb_stats stats;
      |                         ^~~~~

Cc: Kemeng Shi <shikemeng@huaweicloud.com>
Cc: Tejun Heo <tj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/backing-dev.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/backing-dev.c~writeback-support-retrieving-per-group-debug-writeback-stats-of-bdi-fix
+++ a/mm/backing-dev.c
@@ -192,7 +192,6 @@ static int cgwb_debug_stats_show(struct
 	unsigned long background_thresh;
 	unsigned long dirty_thresh;
 	struct bdi_writeback *wb;
-	struct wb_stats stats;
 
 	global_dirty_limits(&background_thresh, &dirty_thresh);
 
_
